### DISCUSSION TITLE
已经开启了全局异常拦截，如何自定义异常拦截返回类型？有了解的大家可以一起交流

### DISCUSSION THREAD

**Discussion by mingyang66** [mingyang66/spring-parent#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mingyang66** [comment]
It's already resolved！

**Comment by Iris-29-L** [comment]
我是想和您联系的猎头，在做数据SDK方向，想和您交个朋友，后续行业动态或者朋友看机会都可以找我
